src/google/protobuf/port_def.inc

```diff
@@ -766,6 +766,8 @@
 #pragma warning(disable : 4073)
 // To silence the fact that we will pop this push from another file
 #pragma warning(disable : 5031)
+// decimal digit terminates octal escape sequence
+#pragma warning(disable: 4125)
```

Comment:
where's the code that triggers this warning?

Reply:
This is caused by the likes of these in several .pb.cc files.
https://github.com/protocolbuffers/protobuf/blob/master/src/google/protobuf/any.pb.cc#L57
https://github.com/protocolbuffers/protobuf/blob/master/src/google/protobuf/descriptor.pb.cc#L925

Reply:
More specifically the warning is thrown in all these places:
```
src\google\protobuf\api.pb.cc(128,3): warning C4125: decimal digit terminates octal escape sequence
src\google\protobuf\api.pb.cc(129,3): warning C4125: decimal digit terminates octal escape sequence
src\google\protobuf\api.pb.cc(130,3): warning C4125: decimal digit terminates octal escape sequence
src\google\protobuf\api.pb.cc(131,3): warning C4125: decimal digit terminates octal escape sequence
src\google\protobuf\api.pb.cc(132,3): warning C4125: decimal digit terminates octal escape sequence
src\google\protobuf\api.pb.cc(133,3): warning C4125: decimal digit terminates octal escape sequence
src\google\protobuf\api.pb.cc(139,3): warning C4125: decimal digit terminates octal escape sequence
src\google\protobuf\compiler\plugin.pb.cc(163,3): warning C4125: decimal digit terminates octal escape sequence
src\google\protobuf\compiler\plugin.pb.cc(165,3): warning C4125: decimal digit terminates octal escape sequence
src\google\protobuf\compiler\plugin.pb.cc(168,3): warning C4125: decimal digit terminates octal escape sequence
src\google\protobuf\compiler\plugin.pb.cc(171,3): warning C4125: decimal digit terminates octal escape sequence
src\google\protobuf\descriptor.pb.cc(1005,3): warning C4125: decimal digit terminates octal escape sequence
src\google\protobuf\descriptor.pb.cc(1017,3): warning C4125: decimal digit terminates octal escape sequence
src\google\protobuf\descriptor.pb.cc(1024,3): warning C4125: decimal digit terminates octal escape sequence
src\google\protobuf\descriptor.pb.cc(1027,3): warning C4125: decimal digit terminates octal escape sequence
src\google\protobuf\descriptor.pb.cc(1029,3): warning C4125: decimal digit terminates octal escape sequence
src\google\protobuf\descriptor.pb.cc(1033,3): warning C4125: decimal digit terminates octal escape sequence
src\google\protobuf\descriptor.pb.cc(1038,3): warning C4125: decimal digit terminates octal escape sequence
src\google\protobuf\descriptor.pb.cc(1041,3): warning C4125: decimal digit terminates octal escape sequence
src\google\protobuf\descriptor.pb.cc(1045,3): warning C4125: decimal digit terminates octal escape sequence
src\google\protobuf\descriptor.pb.cc(1048,3): warning C4125: decimal digit terminates octal escape sequence
src\google\protobuf\descriptor.pb.cc(1051,3): warning C4125: decimal digit terminates octal escape sequence
src\google\protobuf\descriptor.pb.cc(1053,3): warning C4125: decimal digit terminates octal escape sequence
src\google\protobuf\descriptor.pb.cc(1058,3): warning C4125: decimal digit terminates octal escape sequence
src\google\protobuf\descriptor.pb.cc(1065,3): warning C4125: decimal digit terminates octal escape sequence
src\google\protobuf\descriptor.pb.cc(1070,3): warning C4125: decimal digit terminates octal escape sequence
src\google\protobuf\descriptor.pb.cc(927,3): warning C4125: decimal digit terminates octal escape sequence
src\google\protobuf\descriptor.pb.cc(928,3): warning C4125: decimal digit terminates octal escape sequence
src\google\protobuf\descriptor.pb.cc(932,3): warning C4125: decimal digit terminates octal escape sequence
src\google\protobuf\descriptor.pb.cc(933,3): warning C4125: decimal digit terminates octal escape sequence
src\google\protobuf\descriptor.pb.cc(934,3): warning C4125: decimal digit terminates octal escape sequence
src\google\protobuf\descriptor.pb.cc(935,3): warning C4125: decimal digit terminates octal escape sequence
src\google\protobuf\descriptor.pb.cc(936,3): warning C4125: decimal digit terminates octal escape sequence
src\google\protobuf\descriptor.pb.cc(937,3): warning C4125: decimal digit terminates octal escape sequence
src\google\protobuf\descriptor.pb.cc(938,3): warning C4125: decimal digit terminates octal escape sequence
src\google\protobuf\descriptor.pb.cc(939,3): warning C4125: decimal digit terminates octal escape sequence
src\google\protobuf\descriptor.pb.cc(941,3): warning C4125: decimal digit terminates octal escape sequence
src\google\protobuf\descriptor.pb.cc(942,3): warning C4125: decimal digit terminates octal escape sequence
src\google\protobuf\descriptor.pb.cc(943,3): warning C4125: decimal digit terminates octal escape sequence
src\google\protobuf\descriptor.pb.cc(944,3): warning C4125: decimal digit terminates octal escape sequence
src\google\protobuf\descriptor.pb.cc(945,3): warning C4125: decimal digit terminates octal escape sequence
src\google\protobuf\descriptor.pb.cc(946,3): warning C4125: decimal digit terminates octal escape sequence
src\google\protobuf\descriptor.pb.cc(948,3): warning C4125: decimal digit terminates octal escape sequence
src\google\protobuf\descriptor.pb.cc(949,3): warning C4125: decimal digit terminates octal escape sequence
src\google\protobuf\descriptor.pb.cc(950,3): warning C4125: decimal digit terminates octal escape sequence
src\google\protobuf\descriptor.pb.cc(951,3): warning C4125: decimal digit terminates octal escape sequence
src\google\protobuf\descriptor.pb.cc(952,3): warning C4125: decimal digit terminates octal escape sequence
src\google\protobuf\descriptor.pb.cc(955,3): warning C4125: decimal digit terminates octal escape sequence
src\google\protobuf\descriptor.pb.cc(956,3): warning C4125: decimal digit terminates octal escape sequence
src\google\protobuf\descriptor.pb.cc(959,3): warning C4125: decimal digit terminates octal escape sequence
src\google\protobuf\descriptor.pb.cc(962,3): warning C4125: decimal digit terminates octal escape sequence
src\google\protobuf\descriptor.pb.cc(963,3): warning C4125: decimal digit terminates octal escape sequence
src\google\protobuf\descriptor.pb.cc(968,3): warning C4125: decimal digit terminates octal escape sequence
src\google\protobuf\descriptor.pb.cc(980,3): warning C4125: decimal digit terminates octal escape sequence
src\google\protobuf\descriptor.pb.cc(982,3): warning C4125: decimal digit terminates octal escape sequence
src\google\protobuf\descriptor.pb.cc(983,3): warning C4125: decimal digit terminates octal escape sequence
src\google\protobuf\descriptor.pb.cc(985,3): warning C4125: decimal digit terminates octal escape sequence
src\google\protobuf\descriptor.pb.cc(989,3): warning C4125: decimal digit terminates octal escape sequence
src\google\protobuf\descriptor.pb.cc(990,3): warning C4125: decimal digit terminates octal escape sequence
src\google\protobuf\descriptor.pb.cc(992,3): warning C4125: decimal digit terminates octal escape sequence
src\google\protobuf\descriptor.pb.cc(993,3): warning C4125: decimal digit terminates octal escape sequence
src\google\protobuf\descriptor.pb.cc(994,3): warning C4125: decimal digit terminates octal escape sequence
src\google\protobuf\descriptor.pb.cc(997,3): warning C4125: decimal digit terminates octal escape sequence
src\google\protobuf\struct.pb.cc(127,3): warning C4125: decimal digit terminates octal escape sequence
src\google\protobuf\struct.pb.cc(129,3): warning C4125: decimal digit terminates octal escape sequence
src\google\protobuf\struct.pb.cc(130,3): warning C4125: decimal digit terminates octal escape sequence
src\google\protobuf\struct.pb.cc(131,3): warning C4125: decimal digit terminates octal escape sequence
src\google\protobuf\struct.pb.cc(134,3): warning C4125: decimal digit terminates octal escape sequence
src\google\protobuf\struct.pb.cc(135,3): warning C4125: decimal digit terminates octal escape sequence
src\google\protobuf\struct.pb.cc(137,3): warning C4125: decimal digit terminates octal escape sequence
src\google\protobuf\type.pb.cc(188,3): warning C4125: decimal digit terminates octal escape sequence
src\google\protobuf\type.pb.cc(190,3): warning C4125: decimal digit terminates octal escape sequence
src\google\protobuf\type.pb.cc(191,3): warning C4125: decimal digit terminates octal escape sequence
src\google\protobuf\type.pb.cc(192,3): warning C4125: decimal digit terminates octal escape sequence
src\google\protobuf\type.pb.cc(193,3): warning C4125: decimal digit terminates octal escape sequence
src\google\protobuf\type.pb.cc(194,3): warning C4125: decimal digit terminates octal escape sequence
src\google\protobuf\type.pb.cc(195,3): warning C4125: decimal digit terminates octal escape sequence
src\google\protobuf\type.pb.cc(198,3): warning C4125: decimal digit terminates octal escape sequence
src\google\protobuf\type.pb.cc(212,3): warning C4125: decimal digit terminates octal escape sequence
src\google\protobuf\type.pb.cc(213,3): warning C4125: decimal digit terminates octal escape sequence
src\google\protobuf\type.pb.cc(214,3): warning C4125: decimal digit terminates octal escape sequence
src\google\protobuf\type.pb.cc(218,3): warning C4125: decimal digit terminates octal escape sequence
src\google\protobuf\type.pb.cc(219,3): warning C4125: decimal digit terminates octal escape sequence
```

based on a3e558744